### PR TITLE
ima: list names of the runtime policies

### PR DESCRIPTION
**Pull request by niteeshkd**
Add listruntimepolicy option with 'keylime_tenant -c' to list names of the runtime policies.

### CODE REVIEW

**Comment by THS-on** [comment]
@niteeshkd can you also include an update to the documentation to reflect this new API endpoint?

**Comment by niteeshkd** [comment]
> @niteeshkd can you also include an update to the documentation to reflect this new API endpoint?

@THS-on which document should I update? I can't find any document related to named runtime policies under docs/ .

**Comment by niteeshkd** [comment]
> @niteeshkd REST APIs are documented here: https://github.com/keylime/keylime/blob/master/docs/rest_apis.rst

Thanks! I just updated that doc.

File: keylime/tenant.py [hunk 1]
```diff
@@ -1360,6 +1360,15 @@ def do_show_runtime_policy(self, name: Optional[str]) -> None:  # pylint: disabl
         if response.status_code >= 400:
             raise UserError(response_json)
 
+    def do_list_runtime_policy(self) -> None:  # pylint: disable=unused-argument
```
Comment (THS-on) [inline keylime/tenant.py:1363]: Instead of the pylint ignore, you can declare this a static method.

File: keylime/cloud_verifier_tornado.py [hunk 2]
```diff
@@ -856,62 +856,76 @@ class AllowlistHandler(BaseHandler):
     def head(self) -> None:
         web_util.echo_json_response(self, 400, "Allowlist handler: HEAD Not Implemented")
 
-    def __validate_input(self, method: str) -> Optional[str]:
+    def __validate_input(self, method: str) -> Tuple[bool, Optional[str]]:
```
Comment (THS-on) [inline keylime/cloud_verifier_tornado.py:859]: The change here is a bit confusing, but I also don't see a simpler way to make it nicer.

File: keylime/tenant.py (hunk 1 above)
Comment (niteeshkd) [reply keylime/tenant.py]: Pylint ignore was not needed.

**Comment by maugustosilva** [comment]
The `lint` errors here, very much like #1607, are unrelated to the code changes. We should not hold off this merge because of it.

**Review by maugustosilva — APPROVED** [review]

**Pull request merged**